README.md

```diff
@@ -44,23 +37,57 @@ We have been able to expand on the functionality of the system, but there are st
 - [ ] Scenario 4.2.4 - Display Dashboards *Optional* Scenario
 - [ ] Scenario 4.2.5 - Handle Refund Requests *Optional* Scenario
 
+### Architecture Documents
+	
+* Component diagram for the overall system 
+    * [Train Ticketing System Component Diagram - server side only]()
+* Sequence diagrams for each implemented user scenario 
+    * [Launch Train Ticketing System](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%201%20Launch%20Train%20Ticketing%20System%20Sequence%20Diagram%20SOA.pdf)			
+    * [User Login](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%202%20User%20Login%20Sequence%20Diagram%20SOA.pdf)
+    * [Search Tickets](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%203%20Search%20Tickets%20Sequence%20Digram%20SOA.pdf)
+    * [Select Itinerary](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%204%20Select%20Itinerary%20Sequence%20Diagram%20SOA.pdf)	
+    * [Enter Traveler Info](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%205%20Enter%20Traveler%20Info%20Sequence%20Diagram%20SOA.pdf)
+    * [Enter Payment Info and Checkout](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%206%20Enter%20Payment%20Info%20and%20Checkout%20Sequence%20Diagram%20SOA.pdf)
+    * [Manage Order by Admin](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%207%20Manage%20Order%20By%20Admin%20Sequence%20Diagram%20SOA.pdf)
+    * [Manage Customer Info by Admin](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%208%20Manage%20Customer%20Info%20By%20Admin%20Sequence%20Diagram%20SOA.pdf)	
+    * [Request a Refund](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%209%20Request%20a%20Refund%20Sequence%20Diagram%20SOA.pdf)
+    * [View Order History](https://github.com/bs3265/trainsim-public-group7/blob/master/architecture-documents/Use%20Case%2010%20View%20Order%20History%20Sequence%20Diagram%20SOA.pdf)
```

Comment:
im not sure why but this diagram cuts off the right hand side. not sure if its just me or if its something that would need to be downloaded to view completely.

Reply:
yes, we need a one page version of use case 10